beta/nncf/tensorflow/quantization/quantizers.py

```diff
@@ -376,21 +379,27 @@ def get_quantizer_config(self) -> QuantizerConfig:
             per_channel=self.per_channel
         )
 
-    def get_config(self):
-        qspec = TFQuantizerSpec(self.num_bits,
-                                QuantizationMode.ASYMMETRIC,
-                                signedness_to_force=None,
-                                narrow_range=self.narrow_range,
-                                half_range=self._half_range ,
-                                per_channel=self.per_channel)
-        return deepcopy(qspec.__dict__)
+    def get_config(self, keep_op_name=True):
+        qspec_dict = dict()
+        qspec_dict['num_bits'] = self.num_bits
+        qspec_dict['mode'] = QuantizationMode.ASYMMETRIC
+        qspec_dict['signedness_to_force'] = None
+        qspec_dict['narrow_range'] = self.narrow_range
+        qspec_dict['half_range'] = self._half_range
+        qspec_dict['per_channel'] = self.per_channel
+        config = dict()
+        config['quantizer_spec'] = qspec_dict
+        config['name'] = self.name
+        return deepcopy(config)
```

Comment:
Please, reformat the code in a more pythonic way.

Reply:
Why do you do the deep copy?

Reply:
I already removed it. Please check it